CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ option (WITH_ALEMBIC                        "Build Alembic support"
 option (WITH_OSL                            "Build OSL support"                                     OFF)
 option (WITH_DISNEY_MATERIAL                "Build Disney material"                                 OFF)
 option (WITH_NORMALIZED_DIFFUSION_BSSRDF    "Build Normalized diffusion BSSRDF (patented)"          OFF)
+option (WITH_DIRECTIONAL_DIPOLE_BSSRDF      "Build Directional dipole BSSRDF"                       OFF)
```

Comment:
But truth be told, I wouldn't even bother with a dedicated `#define` unless we actually need to make the release and we haven't fixed the directional dipole code. My hope is that we can fix it before the release so this `#define` would never be required.